scripts/tokenomics-emulation/cere-smart-contract.ts

```diff
@@ -62,11 +68,45 @@ class CereSmartContract {
     const gasLimit = +this.config.network.gas_limit;
     const value = +this.config.network.smart_contract_cere_token_amount_default;
 
-    const {partialFee: txnFee} = await this.cereContract.tx
-      .transfer({ value, gasLimit },destination, amount, 0)
+    const { partialFee: txnFee } = await this.cereContract.tx
+      .transfer({ value, gasLimit }, destination, amount, 0)
       .paymentInfo(sender);
     return txnFee;
   }
+
+  public async deploy(sender: KeyringPair) {
+    console.log(`Deploy smart contract`);
+    const code = new CodePromise(this.api, cere02Abi, cere01Wasm);
+
+    const params = [10000000000, ""];
+    const tx = await code.createBlueprint();
+    return new Promise((res, rej) => {
+      tx.signAndSend(
+        sender,
+        Network.sendStatusCb.bind(this, res, rej)
+      ).catch((err) => rej(err));
+    });
+  }
+
+  /**
+   * Deploy the code on chain
```

Comment:
You have method `deploy` but comment `Deploy code` is here. Please, fix it.

Reply:
Done